src/services/DBService.js

```diff
@@ -36,37 +35,49 @@ module.exports.retrieveAccessToken = (teamId) => {
 	});
 };
 
-module.exports.storeJiraIssueID = (issueID, channel) => {
+module.exports.updateChannelContext = (channel, value, valueType) => {
+    // params set to update issueID
+    var expression = "set issueID = :id, issueIDtimestamp = :ts";
```

Comment:
Probably better to throw an error if we don't recognise the key.

Reply:
Done 🥇